bgls/simulator_test.py

```diff
@@ -210,40 +210,6 @@ def test_run_with_stabilizer_ch_simulator():
     assert result_stabilizer_ch == result_state_vector
 
 
-def test_run_with_stabilizer_ch_simulator_near_clifford():
```

Comment:
Why is this removed?

Reply:
Digging into this more I am a bit confused. Shouldn't we expect the sample outcomes to actually differ somewhat since act_on_near_clifford only approximates the T gates? If I run it on wider randomized circuits the results don't necessarily match.

Reply:
Given non-Clifford `U` the expansion  is `U = \sum_{i = 1}^{N} a_i C_i` where `a_i` are real and `C_i` are Clifford, correct? I believe this is equality if you keep all `N` terms. If the expansion is approximate or if you keep less than `N` terms then there's of course approximation error.

But in any case comparing sampled bitstrings is ill defined in this setting. A better test is to check that the expectation value of a set of observables is (approximately) equal.